functions.go

```diff
@@ -353,6 +353,8 @@ var genericMap = map[string]interface{}{
 	"encryptAES":               encryptAES,
 	"decryptAES":               decryptAES,
 	"randBytes":                randBytes,
+	"gzip":                     gzipCompress,
+	"ungzip":                   gzipDecompress,
```

Comment:
Wouldn't "gunzip" be more generally favored than "ungzip"?

Reply:
Hello, done